docs/changelog/overview.mdx

```diff
@@ -12,6 +12,11 @@ This page lists changes to Flare's API.
 Release notes for the Flare Platform can be found on the [product documentation website](https://docs.flare.io/releases).
 </Note>
 
+<Update label="2025-10" description="API - October 2025">
```

Comment:
I don't think this diserves another changelog entry. I feel like the previous one is good enough. We don't need to be as precise as a git log.

Otherwise you can merge with the previous one if you think it really needs to be called out.